File: Part4/C++/Function_FunctionReturnReversed.cpp
#include <stdio.h>
#include <stdlib.h>

int main()
{
    char userInput[ 100 ];
    int enteredInteger;
    
    printf( "Type 1 and press enter.\n" );
    fgets( userInput, sizeof( userInput ), stdin );
    
    // Trying to assign to function name instead of calling function
    atoi = enteredInteger( userInput );
    
    printf( "The user entered the integer %d\n", enteredInteger );
    
    return 0;
}

File: Part4/C#/Function_FunctionReturnReversed.cs
using System;

class Program
{
    static void Main()
    {
        string userInput;
        int enteredInteger;
        
        Console.WriteLine("Type 1 and press enter.");
        userInput = Console.ReadLine();
        
        // Trying to assign to function name instead of calling function
        int.Parse = enteredInteger(userInput);
        
        Console.WriteLine("The user entered the integer {0}", enteredInteger);
    }
}
